### DISCUSSION TITLE
co-design

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/opendatasicilia.it#28] [Idee/Proposte]
Buonasera a tutti/e! Apro questa discussion per raccogliere un po' di idee sul design del nuovo sito di ODS.

- Noi di ODS siamo legati a qualche colore in particolare? Come immaginate la color palette del nuovo sito? 
- Preferiamo uno stile in particolare? Material oriented? (In caso mi linkate qualche idea?)
- Che ne dite di utilizzare [Adobe XD](https://xd.adobe.com/view/c9fc700a-2a14-4765-b529-af35e93fe73b-9caf/) per il prototipo del sito? È uno strumento gratuito molto comodo che offre la possibilità di commentare delle canvas e di accedere in qualche senso ai "metadati" che stanno dietro i colori, i font e quant'altro.

Abbraccio e buon WE ;)

**Comment by aborruso** [comment]
Caro @dennisangemi buona per me l'idea del co-design

Non abbiamo dei colori, abbiamo quel logo, la nostra storia e soprattutto il nostro modo di fare. Su quest'ultimo non so quali siano il il font, il colore, lo stile, ecc.. giusti.

Io vorrei una comunicazione che ci consentisse di:

- comunicare facilmente le cose che facciamo;
- dare la possibilità di vedere le cose fatte nel tempo, con le varie sfaccettature;
- che illustri come unirsi e inviti a farlo;
- che consenta di mantenere il tutto senza grandi sforzi tecnologici, economici e di tempo.

Lo so, non ti ho risposto

**Comment by dennisangemi** [comment]
## Prima versione
Buonasera a tutti/e! Vi propongo una prima versione super abbozzata. Il link è [questo](https://xd.adobe.com/view/c9fc700a-2a14-4765-b529-af35e93fe73b-9caf/) (e potrete utilizzarlo anche in seguito perchè i contenuti si aggiorneranno man mano che apporteremo delle modifiche). 

## I vantaggi dello strumento
https://xd.adobe.com/view/c9fc700a-2a14-4765-b529-af35e93fe73b-9caf/

Come vi anticipavo, questo strumento potrà tornarci molto utile perchè:
- consente di aggiungere **commenti** (facendo un banale login via google)
![image](https://user-images.githubusercontent.com/77018886/144923997-821323fa-190b-4730-a53e-45ae7c3c6f62.png)

- permette di effettuare il **download** delle **risorse** utilizzate nel progetto
![image](https://user-images.githubusercontent.com/77018886/144924174-316de8e8-5ef3-4ab5-83eb-d1799b42c800.png)

- genera il **codice CSS**
![image](https://user-images.githubusercontent.com/77018886/144924244-cc4e43e9-0c07-4eab-91ea-125b58c379a2.png)

- Collegamenti tra le diverse pagine: è possibile fare click su home, chi siamo e blog

## Qualche nota
### Logo
Ovviamente non è ancora la mia proposta definitiva. Mi sono ispirato alle celle di uno spreadsheet (strumento che utilizziamo molto spesso), "aprendole" e inserendo alcuni "pallini" che mi fanno pensare non solo ai dati ma anche ai/lle singoli/e componenti della nostra community in quanto _collegati/e_.

### Illustrazioni
Sono quelle di https://undraw.co/ . Provate ad effettuare una rapida ricerca e ad inviarmi il link qualora doveste trovare qualcosa di interessante. L'illustrazione della home page rimanda immediatamente alla **collaborazione** e alla rappresentazione dei dati tramite **grafici**

Scusate per la confusione di questo messaggio e per la versione ancora **embrionale** che vi sto proponendo ma sono un po' cotto. Vi chiederei di inserire più commenti possibili in modo tale da raccogliere tanti feedback che mi guidino.

Notte ;)

**Reply by dennisangemi** [reply]
Buongiorno! Ho aggiunto nuove pagine e rettificato i colori che adesso rispondono ad una volontà specifica. Trovate tutto nel link indicato in alto.

Questa è la palette che attualmente ho scelto (ovviamente è ancora suscettibile a cambiamenti)
![image](https://user-images.githubusercontent.com/77018886/145231258-df078335-69e3-4a5a-9d5c-f6c64a8d8495.png)

I colori rispondono ad un volontà specifica perchè:
- **blu**: è colore utilizzato dalle PA, da AGID (enti con i quali ci interfacciamo spesso). Mi fa pensare in generale al mondo tech e al nostro mare. Secondo me rappresenta abbastanza bene la nostra mission legata alla trasparenza a cui viene inevitabilmente affiancato il mondo digitale. Qui ho scelto una tonalità tendente al petrolio. 
- **verde**: anche questo è un colore che viene spesso usato quando si realizzano illustrazioni legate al mondo tech
- **panna/sabbia**: non è stato scelto dietro una riflessione profonda, semplicemente si sposa molto bene con il blu (fa pensare alla spiaggia). È abbastanza chiaro e per questo comodo da utilizzare come background (ad esempio per i tag)
![image](https://user-images.githubusercontent.com/77018886/145234185-ab7fd548-8f9d-4152-83ea-5c4929709122.png)

**Reply by dennisangemi** [reply]
Mi piacerebbe da morire ricevere un feedback da parte di @angelogulina . So che è molto impegnato ma se riuscisse a scrollare un po' di pagine mi farebbe contentissimo ;)

**Reply by pigreco** [reply]
sono particolarmente legato a questo logo di OpenDataSicilia:
![image](https://user-images.githubusercontent.com/7631137/145406331-cfec9fdd-4b6d-4a40-8b2f-a7d3c7536781.png)
e mi piacerebbe molto restasse questo.

**Reply by angelogulina** [reply]
Ciao @dennisangemi! Grazie mille per avermi coinvolto, mi sento (sinceramente) onorato.

Ti ho lasciato alcuni commenti. In linea di massima, la proposta mi sembra molto ben fatta, minimal e lineare. Un ottimo punto di partenza che, a parte i miei commenti ovviamente, non modificherei tantissimo. Preoccupati di pubblicare il sito e poi potrai fare le modifiche man mano.

Bisogna preparare un design per mobile. È utile a capire come utilizzare spazi e come organizzare le info nella pagina. Non possiamo escludere le utenti che arrivano da quel medium.

Bel lavoro!

**Comment by aborruso** [comment]
Aggiungo una richiesta/proposta.

Nella proposta di nuova interfaccia, nei limiti dei tempi e ispirazione di tutte/i, ce ne dovrebbe stare anche una che mantiene il logo.

Perché diamo per possibile ovviamente il cambio del logo, ma se non è tra le alternative è già una scelta fatta.
E questa scelta ancora non c'è stata.

Spero di essermi spiegato.

Grazie

**Reply by cirospat** [reply]
a me piace l'attuale logo. Perché?
Lo abbiamo vissuto con passione e sentimento. Scola sudore. Ci identifica in tutta Italia.
Non lo vedo come qualcosa da cambiare. 
Questo il mio umile pensiero.

**Reply by dennisangemi** [reply]
> Perché diamo per possibile ovviamente il cambio del logo, ma se non è tra le alternative è già una scelta fatta.
> E questa scelta ancora non c'è stata.

Ciao @aborruso , la mia è una proposta (in stato embrionale) e by definition non è stata fatta nessuna scelta perchè potremmo anche pensare di "bocciarla" per intero per millemila motivi. Con ogni probabilità ho sbagliato ad inserire nel prototipo del sito il nuovo logo perchè questo ha inevitabilmente determinato uno spostamento del focus (dal protitipo del sito al dettaglio del logo). In questa discussion vorrei raccogliere dei suggerimenti e dei feedback riguardanti la struttura del sito. Adesso creo una discussion per il logo in mood tale da creare più ordine e non distogliere il focus.

Grazie mille ;)

**Reply by aborruso** [reply]
> Ciao @aborruso , la mia è una proposta (in stato embrionale) e by definition non è stata fatta nessuna scelta perché potremmo anche pensare di "bocciarla" per intero per millemila motivi. Con ogni probabilità ho sbagliato ad inserire nel prototipo del sito il nuovo logo perchè questo ha inevitabilmente determinato uno spostamento del focus (dal protitipo del sito al dettaglio del logo). In questa discussion vorrei raccogliere dei suggerimenti e dei feedback riguardanti la struttura del sito. Adesso creo una discussion per il logo in mood tale da creare più ordine e non distogliere il focus.

Ciao @dennisangemi non hai sbagliato nulla. Io ho commentato sul logo qui, perché qui è emersa la discussione.
Continueremo allora nella nuova.
Scusa me, se ho fatto confusione.

**Comment by gbvitrano** [comment]
Grazie :-)  solo una domanda... se posso... premesso che il logo può essere migliorato, modificato se è il caso anche stravolto. Mi chiedo da dove nasce l'esigenza di modificarlo? L'attuale logo non comunica il mostro modo di essere?

**Reply by dennisangemi** [reply]
Ciao @gbvitrano , forse sta passando il messaggio "Dennis odia l'attuale logo di ODS" ma in realtà non è così. Dato che stiamo creando una nuova veste al nostro sito (e in un certo senso anche alla nostra immagine) mi è venuto naturale chiedere la possibilità di presentare una proposta per un nuovo logo. Manterrei questa discussion per suggerimenti/feedback/proposte relativi al design del sito e creerei una nuova discussion per parlare del logo in dettaglio.

**Reply by gbvitrano** [reply]
Ciao @dennisangemi stiamo parlando di migliorare e rendere più fruibile il sito di ODS ovvero di fare un lavoro di Restyling e non un Rebranding come ho letto in qualche commento. Restyling e Rebranding sono cose molto diverse da non confondere. 
Sentiti libero di proporre tutto ciò che vuoi, ma il logo di ODS per me in questo momento ed in questa fase non è in discussione. Quel logo che tu vuoi cambiare... è la storia di ODS 

Fai tutti gli studi che ritieni opportuni, mostraci come si passa dal logo A a Logo B e parliamone pure :-)

Ps... Rebranding mi fa venire l'orticarica... :-)

**Reply by cirospat** [reply]
aperta Discussion sul Logo ODS:
https://github.com/opendatasicilia/opendatasicilia.it/discussions/33

**Comment by gabacode** [comment]
Ho messo su una [branch](https://github.com/opendatasicilia/opendatasicilia.it/tree/dev) con un sito iniziale che seguirà gli aggiornamenti di questa Discussion.

**Reply by dennisangemi** [reply]
Grazie Gab

**Comment by pigreco** [comment]
> * Noi di ODS siamo legati a qualche colore in particolare? Come immaginate la color palette del nuovo sito?

che io sappia no, nessun colore in particolare, e immagino un color palette semplice con pochi colori; per me il contenuto deve colpire e non i colori.

> * Preferiamo uno stile in particolare? Material oriented? (In caso mi linkate qualche idea?)

non capisco la domanda, io uso WP dal 2015 ed ho usato solo un tema perché sono pigro.

> * Che ne dite di utilizzare [Adobe XD](https://xd.adobe.com/view/c9fc700a-2a14-4765-b529-af35e93fe73b-9caf/) per il prototipo del sito? È uno strumento gratuito molto comodo che offre la possibilità di commentare delle canvas e di accedere in qualche senso ai "metadati" che stanno dietro i colori, i font e quant'altro.

ho visto la tua demo e sembra carina, ma non capisco la sezione che si apre sulla destra, a che serve? 

in generale per me va bene tutto e ti ringrazio per il tempo che ci dedichi.

**Reply by dennisangemi** [reply]
Ciao @pigreco , grazie per le risposte!

>non capisco la sezione che si apre sulla destra, a che serve?
- La prima icona serve per inserire dei commenti (ti invito a farlo)
- La seconda icona serve per accedere alle risorse utilizzate nel prototipo (file svg, font, ecc)
- La terza icona serve per accedere ad altri contenuti che non so se utilizzeremo. Ad esempio sarebbe possibile definire un colore/font e assegnargli un nome

**Comment by dennisangemi** [comment]
Domanda per @gabacode : Se si inserisce una label ad un'immagine durante la stesura di un articolo lato wp, c'è modo in gatsby di applicare ad essa uno stile diverso rispetto al corpo dell'articolo? Vorrei solo capire se è possibile o meno per procedere al design di un'eventuale label

**Reply by gabacode** [reply]
@dennisangemi Si, in entrambi i casi basta aggiungere una regola al CSS per il selector `label` sia lato WP che Gatsby, a seconda di come la si vuole. Basta che l'elemento sia wrappato in tags `<label>` o che aggiunga una classe CSS con un nome particolare. (es class="label")

**Reply by dennisangemi** [reply]
Perfetto, grazie mille

**Reply by gabacode** [reply]
Nel caso di Wordpress mi sembra di vedere che quando aggiungi una label tramite blocks, la tag che crea sia `<figcaption>`, quindi in caso non c'è bisogno di aggiungere tags o classi su wordpress, e da Gatsby si targetizza solo quella.

**Comment by dennisangemi** [comment]
Che ne dite di fissare degli standard per le proporzioni delle immagini da caricare come copertina e anteprima dei blogpost? 
- Le anteprime potrebbero essere quadrate
- Le copertine rettangolari (proporzioni?)

**Reply by dennisangemi** [reply]
Anteprima:
![image](https://user-images.githubusercontent.com/77018886/145904083-0161320d-3f8e-44a5-ab57-828fa621d3ed.png)

Copertina:
![image](https://user-images.githubusercontent.com/77018886/145904131-90fe907e-ff3e-46e0-8099-7e0baed392af.png)
